reV/batch/batch.py

```diff
@@ -167,6 +179,56 @@ def _make_job_tag(self, set_tag, arg_comb):
 
         return job_tag
 
+    def _clean_arg_comb_tag(self, set_tag, arg_comb):
+        """Clean a dictionary of arg combinations for a single job by removing
+        any args that only have one value in the current set tag.
+
+        Parameters
+        ----------
+        set_tag : str
+            Optional set tag to prefix job tag.
+        arg_comb : dict
+            Key-value pairs for this argument combination.
+
+        Returns
+        -------
+        tag_arg_comb : str
+            Arg combinations just for making the job tags. This may not have
+            all the arg combinations for the actual job setup.
+        """
+
+        ignore_tags = []
+
+        for arg in arg_comb.keys():
+            all_values = []
+            for batch_set in self._config['sets']:
+                if (batch_set.get('set_tag', '') == set_tag
+                        and arg in batch_set['args']):
+                    all_values += batch_set['args'][arg]
+
+            if len(all_values) <= 1:
+                ignore_tags.append(arg)
+
+        tag_arg_comb = {k: v for k, v in arg_comb.items()
+                        if k not in ignore_tags}
+
+        return tag_arg_comb
+
+    @property
+    def job_table(self):
+        """Get a dataframe summarizing the batch jobs."""
```

Comment:
Is there a need to generate this everytime job_table is called? I.e., can it change?

Reply:
Yeah, this only gets called when all of the batch job directories are made. So if a user has changed the batch permutations the table should be re-created. In any case, I think this is pretty fast compared to all of the other data copying and SLURM job submission that batch does.